core-bundle/src/Resources/contao/dca/tl_form_field.php

```diff
@@ -575,15 +660,17 @@ public function optionImportWizard()
 	 */
 	public function getFields()
 	{
-		$arrFields = $GLOBALS['TL_FFL'];
+		$fields = array();
 
-		// Add the translation
-		foreach (array_keys($arrFields) as $key)
+		foreach ($GLOBALS['TL_FFL'] as $k=>$v)
 		{
-			$arrFields[$key] = $GLOBALS['TL_LANG']['FFL'][$key][0];
+			if ($this->User->hasAccess($k, 'elements'))
```

Comment:
should be `$this->User->hasAccess($k, 'fields')`

Reply:
Fixed in a5719bb369e7719c5b405b1ae4171e582c34b8cd.